canonical-web-and-design/jaas-dashboard
# Switch 'Register Controller' panel to use Vanilla panel

## Done

Switch 'Register Controller' panel to use Vanilla panels and Framer motion animation

Note: Due to the meandering path of this feature branch, this PR should squash and merge all commits into one before landing on master.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/controllers
- Click 'Register new controller'
- See "Register new controller" panel work as expected

Fixes https://github.com/canonical-web-and-design/jaas-dashboard/issues/866,
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1629
